tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -376,7 +375,10 @@ public static int setDateOrder(int dateOrder) {
     // Behavior Control ---------------------------------------------------------
     $BEHAVIOR_ARITHMETIC_ABORT("tajo.behavior.arithmetic-abort", false),
 
-    // ResultSet ---------------------------------------------------------
+    // When inserting empty data into a partitioned table, set to keep existing data.
+    $TABLE_PARTITION_KEPT_EXISTING_DATA_ENABLED("tajo.table-partition.kept-existing-data.enabled", true),
```

Comment:
This also should be updated by my above comment. Also, it would be nice if the config key is changed to 'tajo.partition.overwrite.even-if-no-result', and its default config should be ``false``.